src/topp/IDFileConverter.cpp

```diff
@@ -619,9 +619,13 @@ class TOPPIDFileConverter :
     {
       Size count = 0;
       Int max_hits = getIntOption_("number_of_hits");
-      if (max_hits < 1)
+      if (max_hits == 0)
       {
-        OPENMS_LOG_WARN << "'number of hits' set to less then 1. No peptide hits will be exported!" << endl;
+        OPENMS_LOG_WARN << "'number of hits' set to 0. No peptide hits will be exported! Please enter the number of peptide hits you wish to export. To export all hits enter a negativ number." << endl;
+      }
+      if (max_hits < 0)
```

Comment:
Just make it <= 0.
Avoids extra code and exporting 0 entries is nothing anyone would ever want